### ISSUE TITLE
Jupyter lab terminal does not work

### ISSUE THREAD

**Issue by hellojinwoo** [jupyterlab/jupyterlab#9193]
## Description
<h3> Problem </h3> 

 - When I execute the .py extension file on jupyterlab terminal, another cmd screen shows up.
 - To make it worse, if the .py file does not require inputs, the cmd screen disappears right away.
 - Example is provided below.
<h3> What I would like to do </h3>
 
 - Executing the .py file on the jupyterlab terminal (and seeing the results as well on the terminal, if it is possible) 

<h3> Details </h3>

 - I have made a simple scripts as below, and its file name is `why_cmd.py` 

       for i in range(1,6):
           print(i)

 - It works perfectly fine when I used cmd to run this `why_cmd.py`, as shown in the picture below.

[![enter image description here][1]][1]

 - However, when I run this file on the jupyterlab terminal, it shows 2 weird behaviors
   1) I need to run files with the command `.\why_cmd.py` instead of `why_cmd.py`
   2) The code is executed on another cmd screen. And I cannot even see the results on the cmd screen because it turns off immediately.

[![enter image description here][2]][2]

  [1]: https://i.stack.imgur.com/5tnJA.png
  [2]: https://i.stack.imgur.com/CZOgd.png

## Context

- Operating System and version: Windows 10 
- Browser and version: Naver Whale Version 2.8.107.16 (64-bit)
- JupyterLab version: Version 2.1.5

**Comment by dhirschfeld** [comment]
The second shell is PowerShell. You can configure which shell gets used for JupyterLab by setting a `SHELL` environment variable:
```
set SHELL=cmd.exe
```
...and then restarting JupyterLab from that (cmd.exe) terminal.

That may or may not fix your problem.

You could also try `python why_cmd.py`. Relying on opaque registry settings to determine what Python to execute a script with isn't great practice and likely to result in problems/confusion.

**Comment by hellojinwoo** [comment]
> The second shell is PowerShell. You can configure which shell gets used for JupyterLab by setting a `SHELL` environment variable:
> 
> ```
> set SHELL=cmd.exe
> ```
> 
> ...and then restarting JupyterLab from that (cmd.exe) terminal.
> 
> That may or may not fix your problem.
> 
> You could also try `python why_cmd.py`. Relying on opaque registry settings to determine what Python to execute a script with isn't great practice and likely to result in problems/confusion.

Thank you @dhirschfeld for your answer. 

I have tried `python why_cmd.py` and `python3 why_cmd.py` but both of them do not work. It is the same thing as `why_cmd.py`.
Can you kindly tell me where to set the `SHELL` environment? It doesn't seem to be in the Jupyter Lab's `setting` tab.

**Comment by dhirschfeld** [comment]
You can set a user-level environment variable as described in:
https://www.techjunkie.com/environment-variables-windows-10/

If you then start a new jupyterlab seession (from a new terminal)  it should use the chosen `cmd.exe` `SHELL`

**Comment by hellojinwoo** [comment]
@dhirschfeld Thank you really for your helpful comments. I solved the problem!! But let me ask you one last question.

Because I don't know where to add the environment variable, I added 
```
Variable Name : SHELL 
Variable Value : cmd.exe
```
to both sections : User variables for Jinwoo (1) & System Variables (2)

Which area should I add those to?

![env_variable](https://user-images.githubusercontent.com/34431729/96732031-3344f700-13b8-11eb-89b2-e31103b84682.png)
